psp/__init__.py

```diff
@@ -1,7 +1,3 @@
-__all__ = ['Pv','options']
+__all__ = ['Pv', 'options']
 
-from psp.Pv import Pv as PV
-
-from ._version import get_versions
-__version__ = get_versions()['version']
-del get_versions
```

Comment:
one more small thing I noticed: since this versioneer stuff is gone, psp no longer self-reports it's version at the top-level

Reply:
Ah right there's some `version.py` stuff to add that references the generated `_version.py`